calibration/CTDMOG/CGINS-CTDMOG-13436__20161019.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13436,CC_ptempa0,1.486758e+002,
 37-13436,CC_ptempa1,-6.423020e-002,
 37-13436,CC_ptempa2,-3.334423e-007,
-37-13436,CC_wbotc,3.4261e-007,
+37-13436,CC_wbotc,3.426067e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT